index.html

```diff
@@ -8,27 +8,50 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-        <header id="header">
+        <header class="header header_desktop">
             <img class ="image_geometry_1" src="images/image_geometry_1.png"/>
            
             <nav><img class ="ic_menu" src="images/ic_menu.png"/></nav>
 
         </header>
+       
+        <ul class="menu">
+            <li class="links"><a href="#hello">Hello</a></li>
+            <li class="links"><a href="#portfolio">Portfolio</a></li>
+            <li class="links"><a href="#about">About</a></li>
+            <li class="links"><a href="#contact">Contact</a></li>
+         </ul>
+         
+      
+       
 
-        <section id="headline">
+        <section class="headline">
+            <div class="mb-headline">
             <h1 class="profileName">Harshita Jain</h1>
             
             <p class="intro">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me. </p>
             <button class="startbtn">Start collaboration</button>
+            <img class="dotImg2" src="images/image_dots.svg">
+        </div>
+        
+        <div>
+            <img class="geometry_big" src="images/image-geometry_11.svg"/>
+        </div>
```

Comment:
- [ ] Kindly add these as `background-image`s since they're part of the design and they're not part of the content 👍